install.sh

```diff
@@ -33,6 +29,23 @@ if [[ "$SSL_MODE" != "disabled" ]]; then
     cat /server-cert/ca.crt >/var/run/mongodb/tls/ca.crt
 fi
 
+client_pem=/var/run/mongodb/tls/client.pem
+if [[ "$SSL_MODE" != "disabled" ]] && [[ -f "$client_pem" ]]; then
+    user=$(openssl x509 -in "$client_pem" -inform PEM -subject -nameopt RFC2253 -noout)
+    user=$(echo ${user#"subject="})
+    export USER=$user
```

Comment:
I think we should use a variable name that is lesser chance of conflict just in case. `INJECT_USER` or something else?

Reply:
okay